### PR TITLE
doc: Fix typos

### PR DESCRIPTION
**Pull request by polluks**
Fix typos in the documentation.

Signed-off-by: Stefan Haubenthal <polluks@sdf.lonestar.org>

### CODE REVIEW

File: doc/_doxygen/doxygen-awesome.css [hunk 1]
```diff
@@ -150,7 +150,7 @@ html {
     --side-nav-arrow-opacity: 0;
```
Comment (jukkar) [inline doc/_doxygen/doxygen-awesome.css:150]: The changes look ok, you need to tweak the commit message a bit.
Please set the commit message something like
```
doc: Fix typos

Fix typos in the documentation.

Signed-off-by: <add here your email id>
```

**Comment by kartben** [comment]
@polluks Thank you very much for taking the time to send this PR. Any chance you could amend your commit as per @jukkar 's recommendation? You will also need to properly set your author name in the form "First Last".
Thank you!

**Comment by tejlmand** [comment]
@polluks thanks for fixing all those typos.

Regarding the comment by @jukkar, then please note that you only changed the text in the PR title and description, but not in the commit message itself.

You can read more on commit messages here: https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines

To update your current commit, then you can do:
```
$ git commit --amend
```
and then edit the commit message, and once it has been updated, then push your changes to your branch using `git push -f https://github.com/polluks/zephyr HEAD:main`

File: doc/build/dts/intro-scope-purpose.rst [hunk 2]
```diff
@@ -41,5 +41,5 @@ general, if you see a macro that starts with ``DT_`` in a Zephyr source file,
 it's probably a ``devicetree.h`` macro. The generated C header contains macros
 that start with ``DT_`` as well; you might see those in compiler error
 messages. You always can tell a generated- from a non-generated macro:
-generated macros have some lowercased letters, while the ``devicetree.h`` macro
```
Comment (mbolivar-ampere) [inline doc/build/dts/intro-scope-purpose.rst:44]: "lowercased" is not a typo:

> lowercase
> [verb](https://www.merriam-webster.com/dictionary/verb)
> lowercased; lowercasing
>
> [transitive verb](https://www.merriam-webster.com/dictionary/transitive)
> : to print or set in lowercase letters

https://www.merriam-webster.com/dictionary/lowercase

**Review by mbolivar-ampere — CHANGES_REQUESTED** [review]
Thank you! But please drop the 'lowercased' changes from the devicetree docs.

**Comment by jukkar** [comment]
Please squash the commits together, fix the commit message as described before and force push the changes.

**Comment by polluks** [comment]
> Please squash the commits together, fix the commit message as described before and force push the changes.

I'm not fit in rebasing...

**Comment by kartben** [comment]
> > Please squash the commits together, fix the commit message as described before and force push the changes.
> 
> I'm not fit in rebasing...

@polluks this should help https://asciinema.org/a/tcQBUH1UXUi3ZxdPg00vOGUby

**Pull request closed without merging**